docs/customization/tweaking-figures.ipynb

```diff
@@ -77,7 +77,7 @@
    "id": "091f34e3-49e2-492e-b70f-f34161e1cb24",
    "metadata": {},
    "source": [
-    "The location of the ticks, as well as their labels, can also be changed in the same way"
+    "To change the location of the ticks, as well as their labels, we act directly on the Matplotlib axes via the `.ax` property of the figure:"
```

Comment:
Since the method you are giving now is not matplotlib-specific, I think I would restucture the notebook a little.
We could first have a section which says nothing about matplotlib, that would include changing axes labels and setting figure title.

We could then have a section on 'further modifications' or something like that, that would include changing tick labels as well as the stuff on controlling axes placement?
We would also move the sentence from the intro that begins with "Instead of providing keyword arguments for tweaking every aspect of the figures..." into that section.